lti_consumer/translations/en/LC_MESSAGES/text.po

```diff
@@ -621,13 +621,13 @@ msgstr ""
 
 #: static/js/xblock_lti_consumer.js:128
 msgid "OK"
-msgstr ""
+msgstr "kk"
 
 #: static/js/xblock_lti_consumer.js:157
 msgid ""
 "Click OK to have your username and e-mail address sent to a 3rd party "
 "application."
-msgstr ""
+msgstr "yesss"
```

Comment:
I'm not really sure this is within the scope of your work, but these two translations look off.